android/src/main/java/io/ably/flutter/plugin/push/PushActivationEventHandlers.java

```diff
@@ -64,13 +64,21 @@ public void onReceive(Context context, Intent intent) {
       switch (action) {
         case PUSH_ACTIVATE_ACTION:
           callCallbackOnDartSide(PlatformConstants.PlatformMethod.pushOnActivate, errorInfo);
-          returnMethodCallResult(resultForActivate, errorInfo);
-          resultForActivate = null;
+          if (resultForActivate != null){
+            returnMethodCallResult(resultForActivate, errorInfo);
```

Comment:
Let's add an affirming log spew above this also. DEBUG level.

Reply:
https://github.com/ably/ably-flutter/pull/303/commits/0e8bacd633b39eb79ce08df9eb1c35cf19224e67

Reply:
https://github.com/ably/ably-flutter/pull/303/commits/61e868d442c07ec8844f32a75d8ca1cb519a9bfa